deploy/deploy.go

```diff
@@ -48,10 +49,9 @@ data:
 )
 
 var (
-	logOut        = log.StandardLogger().Out
+	logOut        = os.Stderr
```

Comment:
so the logs originally go to stderr until line 523?

shouldn't this line setup the logshim?